src/firebase.js

```diff
@@ -29,17 +28,18 @@ if (window.location.hostname === "localhost") {
 // Add participant data and trial data to db
 const initParticipant = (participantId, studyId, startDate) => {
   // return promise with value true if participant and study id match, false otherwise
-    return db.collection(collectionName)
+  return db.collection(collectionName)
     .doc(studyId)
     .collection('participants')
     .doc(participantId)
     .collection('data')
     .doc(startDate)
-    .set({ start_time: startDate, app_version: window.navigator.appVersion, app_platform: window.navigator.platform, results: []})
-    .then(()=>{
+    .set({ start_time: startDate, app_version: window.navigator.appVersion, app_platform: window.navigator.platform, results: [] })
+    .then(() => {
       return true
     })
     .catch((error) => {
+      console.log(error)
```

Comment:
You could print these errors with `console.error`

```suggestion
      console.error(error)
```

Reply:
Oh nice one, thanks.  I'll change it.